Source/WebCore/css/mediaControls.css

```diff
@@ -152,15 +152,13 @@ video:-webkit-full-page-media::-webkit-media-controls-panel {
     color: inherit !important;
 }
 
-video::-webkit-media-text-track-container {
+video[pseudo="-webkit-media-text-track-container"] {
```

Comment:
This can keep using the pseudo class syntax

Reply:
Done in [3fb18c8](https://github.com/WebKit/WebKit/commit/3fb18c8f7d1167791a66c1904ffbf1a107168d4d).